### ISSUE TITLE
Copyright Year Missing From Footer

### ISSUE THREAD

**Issue by loganfranken** [UCSantaBarbara/webguide#69]
The copyright year is missing from the website's footer

**Comment by loganfranken** [comment]
(From the list in #68)

**Comment by tenken** [comment]
Can someone give me a definitive answer on the Year value with respect to copyright. Oftentimes I see stuff like "&copy; CURRENT_YEAR" and people say it's wrong or something.

I always thought the copyright was from the date somethiing was created (for instance the University of California as an entity) and that you held the copyright from that point onward, or the creation of the piece of content onward.

I'm saying that copyright should be &copy; 1982 if you create the content in 1982. It's held as long you enforce the copyright and your entity continues to exist.

I've heard it's wrong to update the copyright visually every year on a page such as:

```
echo "&copy; " . date('Y');
```

**Comment by tenken** [comment]
For example -- this blog posts explanation of the date value:

http://www.dailyblogtips.com/proper-copyright-notice/
- The date the article was 1st published.
- Or a date range in which the articles were published

Thoughts?

**EDIT:**
Another possible answer regarding the date value:
http://webmasters.stackexchange.com/a/8249

**Comment by garster** [comment]
I had to deal with copyright at my previous company, I talked to our Legal department about it. I found out that it was not a requirement to be protected, it happens automatically. You only have to put it where you want to assert your copyright to the world, which UC Regents do. 

Source: http://www.copyright.gov/circs/circ03.pdf

Also I was told the date (year) would be the last time it was changed/updated. The date range looks ugly to me, and I think is more of a vanity thing, like a site counter. My old legal department would get antsy around February if we had not updated the copyright to the current year. We would explain, we had not updated the site since Dec, so it has not changed. That is when I made it show the current year.

The easiest is to just show the current year always, and having some automated fashion to display the current year makes it just work. I have seen sites with an old copyright in the footer, but the content was obviously updated recently.

TLDR: Copyright notice is not required legally to be enforceable. The year should be when first published, but websites are "first published", over and over every time the content is changed. So just programmatically showing the current year makes it easy, and there are no legal ramifications.

**Comment by loganfranken** [comment]
I think we should just follow the [recommendation on the guide](http://webguide.ucsb.edu/footer/#copyright) itself (which agrees with everything you're all saying):

```
the year the site was created or the year of the last major redesign or version of the website
```

I would argue the GitHub transition is a major overhaul, so it should be `2015`.

**Comment by tenken** [comment]
You beat me to it. I like the idea of the `2015` but I also think it's _prestigious_ to say the WSG was started in `2005` (see about page) or at least `2007` from the Three Tiers issue. I'll make a pull request.

**Comment by fablio-work** [comment]
I am flexible either way.  
I don’t really think of looking at the copyright to check on the age/history of a website.
Largely in part due to the inconsistent usage brought up in this thread.
Having the WSG history on the About the WSG page is adequate for me.

I consider the 2015 copyright appropriate due to the github migration.

Aaron Martin  (CNT)
Earth Research Institute
6809 Ellison Hall
UCSB MS-3060
805-893-8415
aaron@eri.ucsb.edu mailto:aaron@eri.ucsb.edu
